tests/features/common/kie-kieserver-common.feature

```diff
@@ -517,4 +517,31 @@ Feature: Kie Server common features
       | variable                            | value                          |
       | KIE_SERVER_CONTAINER_DEPLOYMENT     | test=org.package:mypackage:1.0 |
     Then container log should contain Using standard EnvVar KIE_SERVER_CONTAINER_DEPLOYMENT: test=org.package:mypackage:1.0
-     And container log should contain INFO Attempting to pull dependencies for kjar 0 with
\ No newline at end of file
+     And container log should contain INFO Attempting to pull dependencies for kjar 0 with
+
+  Scenario: Check if the Kafka integration is enabled
+    When container is started with env
+      | variable                               | value                         |
+      | KIE_SERVER_KAFKA_EXT_ENABLED           | true                          |
+      | KIE_SERVER_KAFKA_EXT_BOOTSTRAP_SERVERS | localhost:9092                |
+      | KIE_SERVER_KAFKA_EXT_CLIENT_ID         | app                           |
+      | KIE_SERVER_KAFKA_EXT_GROUP_ID          | jbpm-consumer                 |
+      | KIE_SERVER_KAFKA_EXT_ACKS              | 2                             |
+      | KIE_SERVER_KAFKA_EXT_MAX_BLOCK_MS      | 2000                          |
+      | KIE_SERVER_KAFKA_EXT_AUTOCREATE_TOPICS | true                          |
+      | KIE_SERVER_KAFKA_EXT_TOPICS            | person=human,dog=animal   |
+    Then container log should contain -Dorg.kie.server.jbpm-kafka.ext.disabled=false
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.bootstrap.servers=localhost:9092
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.client.id=app
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.group.id=jbpm-consumer
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.acks=2
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.max.block.ms=2000
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.allow.auto.create.topics=true
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.topics.person=human
+    And container log should contain -Dorg.kie.server.jbpm-kafka.ext.topics.dog=animal
+
+  Scenario: Check if the Kafka integration is disabled
+    When container is started with env
+      | variable                       | value   |
+      | KIE_SERVER_KAFKA_EXT_ENABLED   | false    |
```

Comment:
same here;

Reply:
fixed

Reply:
both conditions can be done on the same test.
better merge the two tests and just add the new checks on the first test you've added, wdyt?
same for bats tests.

Reply:
Ok, I'll add two mappings and one incomplete in the resultant test